docs/client/maestro.md

```diff
@@ -4,89 +4,317 @@ title: MAESTRO
+
+Runway assignments can be manually overridden, however avoid changing the runway less than 15 minutes prior to the `ETA_FF` to prevent disrupting the stable sequence. If a runway change is required during this period, coordinate with the flow controller.
+
+### Exchanging Flights
+
+The swap flights function allows two flights to exchange positions in the sequence. This should be used to reflect the actual sequence when tactical control actions have resulted in a different landing order than planned.
+
+Do not swap flights until they are Stable, Super Stable, or Frozen. Unstable flights are continuously being re-sequenced, and swapping them may result in immediate re-sequencing that negates the swap.
+
+To swap two flights:
+
+1. Left-click the first flight to select it
+2. Left-click the second flight
+3. Confirm the swap when prompted
+
+The two flights will exchange their `STA` and runway assignments.
+
+## Enroute Responsibilities
+
+### Issuing Runway Assignments
+
+Issue the arrival runway to the flight based on the Maestro allocation. This can be done while the flight is unstable.
+
+If a runway other than the assigned runway is required, advise the flow controller.
+
+### Unstable Flights
+
+Do not issue delay instructions to unstable flights. Unstable flights are continuously being re-sequenced and their delay requirements may change rapidly. Wait until the flight becomes stable before issuing delay instructions.
+
+### Estimate Accuracy
+
+Ensure the `ETA_FF` is accurate at least 10 minutes prior to the feeder fix. If the system estimate is inaccurate, use the Change ETA_FF function (right-click the flight, select Change ETA_FF) to manually adjust it.
+
+Remove any PETOs (Pilot Estimated Time Over) that may affect the accuracy of the system estimates.
+
+### Re-routing
+
+When re-routing a flight via a new feeder fix, use the Recompute function (right-click the flight, select Recompute) after the re-route has been processed. This will recalculate the flight's feeder fix, estimates, and position in the sequence.
+
+### High Priority Flights
+
+For high priority flights (generally used during busy events like WorldFlight), use Manual Delay `00` to ensure the flight receives minimal delay. If a re-route is required, perform the re-route before selecting Manual Delay `00`.
+
+### Feeder Fix Crossing
+
+Ensure flights cross the feeder fix at the `STA_FF`. Use speed control, vectors, or holding to manage the aircraft's arrival time at the feeder fix.
+
+Advise TMA when a flight is more than 1 minute early or 30 seconds late.
+
+#### Crossing Restrictions
+
+In real-world operations, ATC issues aircraft with a time to cross the feeder fix. However, this can be impractical on the VATSIM network as some pilots' clocks may be out of sync, or are otherwise unable to comply with the instruction.
+Prefer using speed control, vectors, and holding. Use your best judgement if you believe the pilot can comply with a time restriction.
```

Comment:
During a trial run of the plugin, we issued pilots with crossing restrictions and the majority of them were able to comply. I believe the previous consensus was that we shouldn't use this technique, though it's been used quite successfully more recently.
Happy to re-word this section if that's still the case. No strong opinions either way.

Reply:
This is interesting to hear!

When you say "able to comply", is this in the sense that they accepted the time that ATC gave them? Or actually successfully met the time (within ±60 secs) that ATC gave them?

I very rarely control on the network these days, but in my limited experience with crossing times, even if pilot proficiency is 100% perfect, it is simply impossible to get all ATCs and Pilots with the exact same system time to make this work. And even a 2 minute difference in system time would obviously be catastrophic for the sequence.

But interested to hear how it went practically!